frontend/cli/cmd_serve.go

```diff
@@ -404,7 +448,46 @@ func waitForControllerOnline(ctx context.Context, startupTimeout time.Duration,
 	}
 }
 
-func (s *serveCmd) isRunning(ctx context.Context, client ftlv1connect.ControllerServiceClient) bool {
+type pingableClient interface {
```

Comment:
This interface already exists [here](https://github.com/TBD54566975/ftl/blob/1ddd5830699bf866c66e79f3c2df20c898969117/internal/rpc/rpc.go#L90).